### DISCUSSION TITLE
Faceswap not Reactor based?

### DISCUSSION THREAD

**Discussion by MrFranco2** [lllyasviel/Fooocus#2238] [Q&A]
Hi developers, Thanks for this awesome software, using it and A1111 beginner skills. I'm using foocus 2.1.865 version, noted that the face swap feature doesnt match the likeness of the image prompt. setting: stop at 0.9 weight 0.75. Comparing with Reactor extension in A1111 the face swap matches the face more. what tweaks I can do in focus to improve face matching? Thanks for the help.
![Screenshot 2024-02-11 204400](https://github.com/lllyasviel/Fooocus/assets/117552845/c0884504-4b37-4165-a7b8-c7ad5b803e10)

**Comment by xhoxye** [comment]
stop at 1,weight 1 might be a little bit more like
